File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int small = 0;
        int big = numbers.length-1;
        int diffRight=0;
        int diffLeft=0;
        while(small!=big){
            if((numbers[small]+numbers[big])==target){
                return new int[]{small+1,big+1};
            }
            if(numbers[small]+numbers[big]<target){
                small++;
               
            }
            if(numbers[small]+numbers[big]>target){
                big--;
            }
        }
        return new int[]{};
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        complement = {}

        for i in range(len(nums)):
            if(not (complement.get(nums[i]) == None)):
                return [complement.get(nums[i]),i]
            complement[target-nums[i]] = i

        return None;


        